Cesium3DTilesSelection/test/TestTilesetContentManager.cpp

```diff
@@ -476,6 +476,7 @@ TEST_CASE("Test tile state machine") {
     pMockedLoader->mockCreateTileChildren = {{}, TileLoadResultState::Success};
```

Comment:
Is it possible to design a unit test to catch this case? (Maybe in the implicit loader tests?)